Extensions/PlatformBehavior/tests/platformerobjectruntimebehavior.spec.js

```diff
@@ -996,6 +1102,108 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
       climbLadder(2);
     });
 
+    it('can grab a ladder while on the descending phase of a jump', function () {
+      // Need a bigger ladder
+      ladder.getHeight = function () {
+        return 300;
+      };
+      ladder.setPosition(30, -10 - ladder.getHeight());
+
+      object.setPosition(30, -32);
+      // Ensure the object falls on the platform
+      fallOnPlatform(10);
+
+      // Jump
+      object.getBehavior('auto1').simulateJumpKey();
+      for (let i = 0; i < 19; ++i) {
+        jumpAndAscend(1);
+      }
+
+      // starting to going down
+      object.getBehavior('auto1').simulateLadderKey();
+      stayOnLadder(1);
+      expect(object.getBehavior('auto1').isJumping()).to.be(false);
+
+      stayOnLadder(10);
+      climbLadder(2);
+    });
+
+    //TODO Probably a bug, remove the skip when it's fixed
+    it.skip("can't grab a ladder while on the ascending phase of a jump", function () {
```

Comment:
Well it's not a bug in the current situation right, you should be able to grab a ladder while in the ascending phase of a jump without any problem?

Reply:
I mean, this kind of things allow to do the infinite "jump+ladder" trick, we'll fix this with a new check that is preventing to grab ladder either too soon after a jump or in certain conditions, but I think it's fine to not consider this a bug for now :)

Reply:
I removed it, I'm keeping it under my hat if it can help to write a test when someone find a solution.